defold-rive/editor/src/rive.clj

```diff
@@ -206,6 +215,9 @@
         animations (map (fn [index] (plugin-get-animation handle index)) indices)]
     (concat [""] animations)))
 
+
+(set! *warn-on-reflection* false)
```

Comment:
Since Clojure doesn't understand that the classes are loaded dynamically, we silence the spam warnings like this.